src/player.ts

```diff
@@ -2006,6 +2007,23 @@ export default class Player extends FakeEventTarget {
     }
   }
 
+  /**
+   * In iframe embed add title element with the entry name
+   * @returns {void}
+   * @private
+   */
+  private _addTitleOnIframe(): void {
+    if (window.self !== window.top) {
+      const head = Utils.Dom.getElementBySelector('head');
+      let title = head.querySelector('title');
+      if (!title){
+        title = Utils.Dom.createElement('title');
+        head.appendChild(title);
+      }
+      title.innerHTML = this._sources.metadata.name ? this._sources.metadata.name : '';
```

Comment:
what if metadata is not set ?

Reply:
what if the title is not empty ? meaning, if the page containing the player already set an iframe text ?